### ISSUE TITLE
toast is error in IE 11

### ISSUE THREAD

**Issue by phamvanlinh20111993** [fkhadra/react-toastify#297]
i using toast in IE 10 and i have some problems with this.
SCRIPT438: Object doesn't support property or method 'parseFloat', please help !!! thanks

**Comment by goenning** [comment]
You could either use polyfill or rollback to 4.4.3

**Comment by fkhadra** [comment]
Hello,

I just noticed that ie don't support `Number.parseFloat`. I could use the one on window instead.
BTW, the polyfill on mdn is dead simple:
```js
if (Number.parseFloat === void 0) {
    Number.parseFloat = parseFloat;
}
```

I'll push a fix in the hour.

**Comment by fkhadra** [comment]
@phamvanlinh20111993 the fix is available with the version 4.5.2
